packages/common-integration-tests/src/cache/sorted-set.ts

```diff
@@ -109,6 +110,13 @@ export function runSortedSetTests(
         });
         expect((changeResponse as IResponseSuccess).is_success).toBeTrue();
         await sleep(timeout * 1000);
+        const getItemTTLResponse = await cacheClient.itemGetTtl(
+          integrationTestCacheName,
+          sortedSetName
+        );
+        expectWithMessage(() => {
+          expect(getItemTTLResponse).toBeInstanceOf(CacheItemGetTtl.Miss);
+        }, `expected MISS but got ${getItemTTLResponse.toString()}`);
```

Comment:
will this error message include the string showing us what the remaining TTL is?

If part of the goal here is to give us more debug info, it might also be worthwhile to cal itemGetTtl after the first write and log what the ttl is there. maybe even call it before the sleep too and log it :)